terraform/gold-eb75ad-team/dashboards.tf

```diff
@@ -297,14 +297,47 @@ resource "sysdig_monitor_dashboard" "general_pod_performance" {
     height      = 4
     type        = "timechart"
     name        = "Memory used by pod vs memory request keycloak"
-    description = "Keycloak memory used by pods compared to the requested memory"
+    description = "Keycloak memory used by pods compared to the requested memory factor of 100 added to fix dispaly"
```

Comment:
Looks great to me,  just a typo on "display" here and the description below

Reply:
fixed